### ISSUE TITLE
Inspections: Cloudflare credentials should not be mandatory

### ISSUE THREAD

**Issue by lxop** [VladRassokhin/intellij-hcl#200]
Cloudflare credentials can be provided as environment variables as an alternative to passing them to the provider block: https://www.terraform.io/docs/providers/cloudflare/index.html#argument-reference
As such, omitting the email and token fields should not be marked as a problem.

### Prerequisites
 - [X] Ensure you have latest version of plugin installed
 - [X] Search for possible issue duplicates

### Installation details
 - [X] IDE version (`Help->About->Copy to Clipboard`)
IntelliJ IDEA 2018.3.5 (Community Edition)
Build #IC-183.5912.21, built on February 26, 2019
JRE: 1.8.0_152-release-1343-b28 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.3
 - [X] intellij-hcl plugin version (`Settings->Plugins`)
0.6.13
 - [X] Terraform version (`terraform -v`)
Terraform v0.11.10

### Terraform Configuration Files
```hcl
provider "cloudflare" {
}
```

### Exception
```text
Missing required properties: email, token
```

### Expected Behavior
No problems reported by the inspections.

### Actual Behavior
The above problem found by the inspections

### Steps to Reproduce
1. Have a cloudflare provider block
2. Omit credentials because they will be provided by environment variables

**Comment by VladRassokhin** [comment]
Will be available in next release

**Comment by VladRassokhin** [comment]
Released in 0.6.14
